pii_scan.py

```diff
@@ -36,6 +36,12 @@ def analyze_text(text: str, show_supported=False, show_details=False) -> list[st
                            score=0.9)
     uuid_recognizer = PatternRecognizer(supported_entity='UUID',
                                         patterns=[uuid_pattern])
+    
+    banner_id_pattern = Pattern(name='banner_id_pattern',
```

Comment:
Looking at your original requirements it looks to me like you were planning to detect 950

>Actually, I'll include it all so that it can detect whether they include 950 or not!

Did that comment mean you were planning to detect any 9-digit number?

Also in your example of valid banner IDs you had _3456343_ which is 7 digits.  I'm assuming this example in your original requirements was just a mistake?

Reply:
Yes, I meant that it will detect any 9 digit or 6 digit number and what i put in the example as a mistake, I'll fix it.